var/spack/repos/builtin/packages/nfft/package.py

```diff
@@ -25,7 +25,7 @@
 from spack import *
 
 
-class Nfft(AutotoolsPackage):
+class Nfft(Package):
```

Comment:
@citibeth in other words, `AutotoolsPackage` package is NOT used here at all, the installation is still done via custom `install()` as opposed to overriding `def configure_args(self):` which is used in  `AutotoolsPackage`.